.ci/tweet-releasenotes.sh

```diff
@@ -28,22 +28,35 @@ else
   cd ../
 fi
 
-CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
-echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
-
 cd .ci-temp/checkstyle
-LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
-                       | jq ".tag_name")
-echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
+
+curl \
+ https://api.github.com/repos/checkstyle/checkstyle/releases \
+ -o /var/tmp/cs-releases.json
+
+TARGET_RELEASE_NUM=$1
+TARGET_RELEASE_INDEX=$(cat /var/tmp/cs-releases.json | \
+    jq '[.[].tag_name] | to_entries | .[] | \
+    select(.value=="checkstyle-'"$TARGET_RELEASE_NUM"'") | .key')
```

Comment:
please make it like:
```
TARGET_RELEASE_INDEX=$(cat /var/tmp/cs-releases.json | \
    jq "[.[].tag_name] | to_entries | .[] | \
    select(.value==\"checkstyle-$TARGET_RELEASE_NUM\") | .key")
```

I feel uncertain  with jq and combined `''` strings ... it works, but it not sure why, lets avoid this style for now.

Reply:
Changes Done